qlkube/deploy/qlkube/values.yaml

```
@@ -0,0 +1,47 @@
+# Default values for qlkube.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: crownlabs/qlkube
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+deploymentAnnotations: {}
+podAnnotations: {}
+
+securityContext:
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+  runAsUser: 101
+  runAsGroup: 101
+  privileged: false
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+  hostname: qlkube.crownlabs.example.com
+  path: /
+  secret: qlkube-certificate
+
+resources:
+  limits:
+    cpu: 1000m
+    memory: 250Mi
+  requests:
+    cpu: 100m
+    memory: 100Mi
```

```suggestion
    cpu: 1000m
    memory: 500Mi
  requests:
    cpu: 100m
    memory: 250Mi
```

Looking at the stats of the one deployed in the cluster, I would increase a bit the memory